docs/readme.md

```diff
@@ -0,0 +1,61 @@
+
+## Table of contents
+
+1. [Before you begin](#before-you-begin)
+2. [Ways to contribute](#ways-to-contribute)
+3. [Updating compatibility tables on MDN](#updating-compatibility-tables-on-mdn)
+4. [Opening issues and pull requests](#opening-issues-and-pull-requests)
+   1. [Generating data using the Web API Confluence Dashboard](#generating-data-using-the-web-api-confluence-dashboard)
+5. [Getting help](#getting-help)
+
+## Before you begin
+
+The browser-compat-data project (BCD) welcomes contributors of all kinds, but we ask that you keep these guidelines in mind when you're contributing.
+
+The project requires that all contributors follow [Mozilla's code of conduct and etiquette guidelines](/CODE_OF_CONDUCT.md).
+
+This project has [a formal governance document](/GOVERNANCE.md), which describes how various types of contributors work within the project and how decisions are made.
+
+The repository is made available under the terms the [Creative Commons CC0 Public Domain Dedication](/LICENSE). Any contributions must be compatible with its terms. If you're not sure about that, [please ask](#getting-help).
+
+## Ways to contribute
+
+There are many ways you can help improve this repository! For example:
+
+* **Add new compat data**: familiarize yourself with the [schema](https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data.schema.json) and read the [schema docs](https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md) to add new files.
+* **Fix existing compat data**: maybe a browser now supports a certain feature. Yay! If you open a PR to fix a browser's data, it would be most helpful if you include a link to a bug report or similar so that we can double-check the good news.
+* **Fix a bug:** we have a list of [issues](https://github.com/mdn/browser-compat-data/issues),
+or maybe you found your own.
+* **Review a pull request:** there is a list of [PRs](https://github.com/mdn/browser-compat-data/pulls).
+Let us know if these look good to you.
+
+## Updating compatibility tables on MDN
+
+It takes 1-2 weeks for changes in this data to be reflected in MDN's browser compatibility tables. The process is:
+
+1. A pull request is reviewed and merged to master.
+2. A new release of [mdn-browser-compat-data](https://www.npmjs.com/package/mdn-browser-compat-data) is created by MDN staff. This happens every 4-14 days.
+3. A new image of [Kumascript](https://github.com/mdn/kumascript), which includes the BCD release, is built and deployed to production. This happens within a day of the npm package release.
+4. The MDN page using the data is regenerated. For newly converted pages, a staff member switches to the [{{Compat}}](https://github.com/mdn/kumascript/blob/master/macros/Compat.ejs) macro, and re-checks the conversion. For updates to converted pages, a logged-in MDN user [force-refreshes the page](https://en.wikipedia.org/wiki/Wikipedia:Bypass_your_cache#Bypassing_cache) to regenerate it.
```

Comment:
Doesn't need to be in this PR, but a note for later: 
The point about "a staff member switches to the {{Compat}} macro" is now obsolete as all but 2 en-US pages already use the correct compat macro.

I think instead we should point to two scenarios: 
1. For pages that already have a compat table, we should say how to refresh the rendering (and that it now happens monthly automatically).
2. For new pages, a compat macro needs to be added, see https://developer.mozilla.org/en-US/docs/MDN/Contribute/Structures/Compatibility_tables#Inserting_the_data_into_MDN_pages

Also:
 "It takes 1-2 weeks for changes in this data to be reflected in MDN's browser compatibility tables"

I think this should rather be:
 "It takes 2-4 weeks for changes in this data to be rendered in MDN's browser compatibility tables"

Reply:
Making a note of this on the parent issue. Thank you!